### PR TITLE
Added Blackbody manifest

### PR DESCRIPTION
**Pull request by anieuwland**
### Confirming that Blackbody meets all the criteria

I believe I [thoroughly](https://bitbucket.org/nimmerwoner/blackbody/issues/19/make-blackbody-available-for-installation) checked everything and made my application compilable through flatpak-builder without network access.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributible by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the [project](http://blackbody.nimmerfort.eu).
- [x] [I own](http://nimmerfort.eu) the domain used in the application ID ([eu.nimmerfort.blackbody](http://blackbody.nimmerfort.eu)).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. (Not applicable)

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: eu.nimmerfort.blackbody.json [hunk 1]
```diff
@@ -0,0 +1,1040 @@
+{
+  "app-id": "eu.nimmerfort.blackbody",
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args": [],
```
Comment (bilelmoussaoui) [inline eu.nimmerfort.blackbody.json:5]:
```suggestion
```

File: eu.nimmerfort.blackbody.json [hunk 2]
```diff
@@ -0,0 +1,1040 @@
...
+  ],
+  "modules": [
+    {
+      "builddir": true,
+      "buildsystem": "meson",
+      "name": "blackbody",
+      "config-opts": [
+      	"--buildtype=release"
+      ],
+      "sources": [
```
Comment (bilelmoussaoui) [inline eu.nimmerfort.blackbody.json:36]: You could put the automatically generated stuff in a separate file

File: eu.nimmerfort.blackbody.json [hunk 3]
```diff
@@ -0,0 +1,1040 @@
...
+          "type": "git",
+          "url": "https://bitbucket.org/nimmerwoner/blackbody/",
+          "tag": "release-1.0.0",
+          "commit": "28599f0050a0a03c79b9415b4411c434324affbf"
+        }
+      ]
+    }
+  ],
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
```
Comment (bilelmoussaoui) [inline eu.nimmerfort.blackbody.json:1035]:
```suggestion
  "runtime-version": "3.38",
```

**Comment by anieuwland** [comment]
I applied the suggested changes.

Regarding the runtime-version's value, is it generally recommended to use as recent a version as possible or as old as possible? I reasoned old to have the widest available compatibility.

**Comment by barthalion** [comment]
Flatpak will install the required runtime automatically so in general using the latest gives you longer support time.

bot, build eu.nimmerfort.blackbody

**Comment by barthalion** [comment]
Something is wrong. All I get is the icon as in the screenshot below; I've downloaded IR_2412.jpg from thermimage but using `Open` doesn't do anything either. Nothing is logged in the terminal.

image

**Comment by anieuwland** [comment]
> Something is wrong. All I get is the icon as in the screenshot below; I've downloaded IR_2412.jpg from thermimage but using Open doesn't do anything either. Nothing is logged in the terminal.

That's odd. Could you try with [this](https://bitbucket.org/nimmerwoner/blackbody/downloads/flir_e4_3.jpg) or [this](https://user-images.githubusercontent.com/58053690/69399459-0fa69780-0ca3-11ea-80ad-df471850ab07.jpg) file? I'm sure all of the files listed [here](https://bitbucket.org/nimmerwoner/blackbody/downloads/) also work.

I think what's happening is that first photo comes from one of the cameras Blackbody doesn't support yet. I thought I had already implemented better user feedback for that.

**Comment by anieuwland** [comment]
I updated the application to, among other things, show a dialog window if opening fails so the user gets some feedback on why nothing is happening.

Although the app can't open the sample from thermimage, I did test it with 11 other cameras that did work. Support for more models is an on-going process :).

**Comment by bilelmoussaoui** [comment]
bot, build eu.nimmerfort.blackbody

**Pull request closed without merging**